app.js

```diff
@@ -49,7 +49,8 @@ const path = require('path');
 try {
 	require.resolve('sockjs');
 } catch (e) {
-	throw new Error("Dependencies unmet; run npm install");
+	console.error('Dependencies are unmet; run npm install before launching Pokemon Showdown again.');
+	process.exit(1);
```

Comment:
This isn't dependent-friendly. If another project tries to load ``pokemon-showdown/apps`` but it fails, it should result in an error, not in the whole application exiting.